client/state/selectors/get-reader-follows-count.js

```diff
@@ -0,0 +1,12 @@
+/*
+ * Get the count of follows a user has
+ *
+ * @param  {Object}  state  Global state tree
+ * @return {Integer} Follow count
+ */
+const getReaderFollowsCount = state => Math.max(
+	state.reader.follows.itemsCount,
+	Object.keys( state.reader.follows.items ).length,
```

Comment:
The only thing I can think of requires being smart and acknowledging the initial follows sync has completed. Once that happens you can fully rely on the local redux state....but that all feels wrong